meson_options.txt

```diff
@@ -65,7 +65,7 @@ option('libxml', type : 'feature', value: 'auto',
 option('libxslt', type : 'feature', value: 'auto',
   description: 'XSLT support in contrib/xml2')
 
-option('llvm', type : 'feature', value: 'disabled',
+option('llvm', type : 'feature', value: 'auto',
```

Comment:
Why did you change this as part of this PR?

Reply:
Since irlink is called in llvm: https://github.com/anarazel/postgres/blob/30f34f8f6f1dd29b300469afcbba89d619b72b22/meson.build#L375 
I changed that for running irlink at meson build and put in seperate commit.

Reply:
I still don't think we should enable llvm by default. For one there's still a few incompatibilities (as you see in the test). But also more generally, it makes the build a *lot* more expensive. So we shouldn't enable it by default.  How about instead just enabling it explicitly for one of the tasks in .cirrus.yml?